modular_zubbers/code/modules/antagonists/blob/blobstrains/_blobstrain.dm

```diff
@@ -0,0 +1,13 @@
+/datum/blobstrain/
+	var/instant_alert_on_change = FALSE
+
+/datum/blobstrain/reagent/networked_fibers/on_gain()
+	. = ..()
+	if(instant_alert_on_change)
+		announcement_size = 0
```

Comment:
Since you're already making this variable under `/datum/blobstrain`, you should extend its on_gain() proc with the logic